pkg/builtin/manifest.yml

```
@@ -100,6 +100,66 @@ extensions:
               title: Maximum zoom level
               min: 0
               max: 30
+        - id: reTheme
+          title: Publish Theme
+          description: Set your theme.
+          fields:
+            - id: reThemeMode
+              type: string
+              title: Mode
+              defaultValue: preMadeTheme
+              description: Select the theme mode.
+              choices:
+                - key: preMadeTheme
+                  label: Pre-made theme
+                - key: customTheme
+                  label: Custom theme
+            - id: preMadeThemeId
+              type: string
+              title: Pre-made theme
+              description: Select a theme.
+              defaultValue: reEarthDark
+              choices:
+                - key: reEarthDark
+                  label: Re:Earth Dark
+                - key: reEarthLight
+                  label: Re:Earth Light
+                - key: forest
+                  label: Forest
+              availableIf:
+                field: reThemeMode
+                type: string
+                value: preMadeTheme
+            - id: textColor
+              type: string
+              ui: color
+              title: Text color
+              description: Select a color.
+              defaultValue: '#434343'
+              availableIf:
+                field: reThemeMode
+                type: string
+                value: customTheme
+            - id: selectColor
```

```suggestion
            - id: themeSelectColor
```

Since the same field name cannot be used across groups, it is recommended to include a prefix.